web-console/future-openshift/operators-overview/operators-overview.md

```diff
@@ -3,16 +3,15 @@
 This documentation details how operators will be surfaced on the Status page. Rather than exposing CSV objects, the status page should now expose the CRs being managed by identified CSVs in designated groups.
 
 ## Group by: Application
-![image](img/operators8-app.png)
+![image](img/groupby-application.png)
 * In the group by Application view, operator managed CRs will be grouped together with the CSV object name as the group header (AMQ Streams).
 * In this example, the “View operator details” link will bring users to the Installed Operators > AMQ Streams > Overview tab. The “View resources” link will bring users to the Installed Operators > AMQ Streams > Kafka details > Resources tab.
 
 ## Group by: Resource
-![image](img/operators8-resource.png)
+![image](img/groupby-resource.png)
 * In the group by Resource view, CRs will be grouped with other CRs of the same type, with the API type as the group header.
 
 ## Side panel
-![image](img/operators8-overview.png)
+![image](img/sidepanel.png)
 * The side panel will have one tab: Overview.
```

Comment:
Is there a plan to have more in the future? Just wondering if we even need it if not.

Reply:
Good question, I'm not sure. We toyed with the idea of showing resources in a tab but went with the link "View resources" instead based on feedback from the OLM team. I can't recall the exact rationale though. Are you proposing just getting rid of the tab if there's only one view?

Reply:
Yeah exactly. I'm fine with keeping it if we think this is going to grow eventually...just wondering if it's needed since it takes up a fair amount of space otherwise.